functions/artsy/index.js

```diff
@@ -120,6 +128,62 @@ app.intent('AboutIntent', {
     }
 );
 
+app.intent('ShowsIntent', {
+        "slots": {
+            "CITY": "LITERAL"
+        },
+        "utterances": [
+            "{for|} shows {in|around} {-|CITY}"
+        ]
+    },
+    function(req, res) {
+        var city = req.slot('CITY');
+        console.log('app.ShowsIntent: ' + city);
+
+        geocoder.geocode(city)
+            .then(function(geoRes) {
+                geocodeResult = _.first(geoRes);
+                if (_.isEmpty(geoRes)) {
+                    res.say(`Sorry, I couldn't find ${city}. Try again?`);
```

Comment:
Add a log for the city that couldn't be found, I found it very useful to have that info when debugging and testing.